build.sh

```diff
@@ -176,6 +176,18 @@ DISTRIB_RELEASE=\"${LSB_VERSION}\"
 DISTRIB_DESCRIPTION=${SYSTEM_DESC}
 " > /etc/lsb-release
 
+echo '
+NAME="ChimeraOS"
```

Comment:
The build script tries to be agnostic to make it easy to edit everything about the OS in one place and allow anyone to make a fork with a new name easily.

All these values should be defined in the `manifest` file.

For example `$SYSTEM_DESC` is already defined as "ChimeraOS".
And we could use `$SYSTEM_NAME` for `ID`.
The other values will need to be added.